pkg/storage/registrator/agent/agent.go

```diff
@@ -32,7 +33,7 @@ func GetProperty(props map[string]interface{}, t taxonomy.ConnectionType, key st
 	case map[string]interface{}:
 		property := propertyMap[key]
 		if property != nil {
-			return property.(string), nil
+			return fmt.Sprintf("%v", property), nil
```

Comment:
if property == nil, you return `"", nil`, is it correct?
should it be `errors.New("undefined property " + key)` ?
or just 
```golang
func GetProperty(props map[string]interface{}, t taxonomy.ConnectionType, key string) (string, error) {
	propertyMap := props[t]
	if propertyMap == nil {
		return "", errors.New("undefined properties for " + string(t))
	}
	propMap, err := propertyMap.(map[string]interface{})
	if err != true {
		return "", errors.New("undefined property " + key)
	}
	property := propMap[key]
	if property != nil {
		return fmt.Sprintf("%v", property), nil
	}
	return "", errors.New("undefined property " + key)
}
```

Reply:
modified